cl/search/documents.py

```diff
@@ -1070,3 +1081,578 @@ def prepare(self, instance):
+    timestamp = fields.DateField()
+
+    class Django:
+        model = OpinionCluster
+        ignore_signals = True
+
+    def prepare_docketNumber(self, instance):
+        return instance.docket.docket_number
+
+    def prepare_caseName(self, instance):
+        return best_case_name(instance)
+
+    def prepare_caseNameFull(self, instance):
+        return instance.case_name_full
+
+    def prepare_dateFiled_text(self, instance):
+        if not instance.date_filed:
+            return
+
+        if isinstance(instance.date_filed, str):
+            return datetime.strptime(instance.date_filed, "%Y-%m-%d").strftime(
+                "%-d %B %Y"
+            )
+
+        return instance.date_filed.strftime("%-d %B %Y")
+
+    def prepare_dateArgued_text(self, instance):
+        if instance.docket.date_argued:
+            return instance.docket.date_argued.strftime("%-d %B %Y")
+
+    def prepare_dateReargued_text(self, instance):
+        if instance.docket.date_reargued:
+            return instance.docket.date_reargued.strftime("%-d %B %Y")
+
+    def prepare_dateReargumentDenied_text(self, instance):
+        if instance.docket.date_reargument_denied:
+            return instance.docket.date_reargument_denied.strftime("%-d %B %Y")
+
+    def prepare_court_id(self, instance):
+        return instance.docket.court.pk
+
+    def prepare_court(self, instance):
+        return instance.docket.court.full_name
+
+    def prepare_court_citation_string(self, instance):
+        return instance.docket.court.citation_string
+
+    def prepare_judge(self, instance):
+        return instance.judges
+
+    def prepare_panel_names(self, instance):
+        return [judge.name_full for judge in instance.panel.all()]
+
+    def prepare_citation(self, instance):
+        return [str(cite) for cite in instance.citations.all()]
+
+    def prepare_attorney(self, instance):
+        return instance.attorneys
+
+    def prepare_suitNature(self, instance):
+        return instance.nature_of_suit
+
+    def prepare_status(self, instance):
+        return instance.get_precedential_status_display()
+
+    def prepare_proceduralHistory(self, instance):
+        return instance.procedural_history
+
+    def prepare_posture(self, instance):
+        return instance.posture
+
+    def prepare_syllabus(self, instance):
+        return instance.syllabus
+
+    def prepare_sibling_ids(self, instance):
+        return [opinion.pk for opinion in instance.sub_opinions.all()]
+
+    def prepare_panel_ids(self, instance):
+        return [judge.pk for judge in instance.panel.all()]
+
+    def prepare_dateFiled(self, instance):
+        if instance.date_filed is None:
+            return
+
+        if isinstance(instance.date_filed, str):
+            datetime_object = datetime.strptime(
+                instance.date_filed, "%Y-%m-%d"
+            )
+            return midnight_pt(datetime_object)
+
+        return midnight_pt(instance.date_filed)
```

Comment:
I believe we no longer need to convert dates to `midnight_pt `since Elasticsearch allows storing naive `datetime` objects, meaning the date is stored at midnight by default without a timezone.

Reply:
got it! so we just need to remove the `midnight_pt` helper method, right?

Reply:
yeah!